src/components/app/reducers.js

```diff
@@ -0,0 +1,33 @@
+import Cookies from 'cookies-js';
+
+import { SET_COOKIE, REMOVE_COOKIE } from './actionTypes';
+import { EXPIRATION } from './constants';
+
+const initialState = {
+    loggedIn: false,
+};
+
+export default ( state = { ...initialState }, action) => {
+    switch (action.type) {
+    case SET_COOKIE:    {
+        const { token } = action;
+        Cookies.set(token, { expires: EXPIRATION });
+
+        return {
+            ...state,
+            loggedIn: true,
```

Comment:
set the token in the state so that you can have access to it later when you want to remove it from the cookie